src/shapes/polygon.class.js

```diff
@@ -74,7 +74,7 @@
    * @param {Function} [callback] Callback to invoke when an fabric.Path instance is created
    */
   fabric.Polygon.fromObject = function(object, callback) {
-    return fabric.Object._fromObject('Polygon', object, callback, 'points');
+    fabric.Object._fromObject('Polygon', object, callback, 'points');
```

Comment:
can you add return void to this one too?

Reply:
Added